core-services/prow/02_config/kubevirt-ui/kubevirt-plugin/_pluginconfig.yaml

```diff
@@ -0,0 +1,51 @@
+approve:
```

Comment:
You should also add a `_prowplugin.yaml` with Tide config to set up automated merges (ideally copy it from another repositoriy config whose configuration you want mimic)